backend/src/poap/index.js

```diff
@@ -3,12 +3,12 @@ import axios from 'axios';
 class POAP {
 
     static async fetchEvents() {
-        const response = await axios.get('https://api.poap.xyz/events');
+        const response = await axios.get('https://api.poap.tech/events');
```

Comment:
Create a Jira ticket with technical debt to move this URL to an environment variable and add the ticket in this comment.

Reply:
@nicolasdeleon

Reply:
This is now part of the proposed enhancement in https://poap-devs.atlassian.net/browse/PLT-218